### PR TITLE
[polish] Capitalize help on all commands

### PR DESCRIPTION
**Pull request by ipince**
## Summary
TSIA

## How was it tested?
Not tested

## Is this change backwards-compatible?
Yes

### CODE REVIEW

**Review by savil — APPROVED** [review]
heh, we keep going back and forth on this  :)

File: padcli/command/initcmd.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 func initCmd() *cobra.Command {
 	var initCmd = &cobra.Command{
 		Use:   "init [path]",
-		Short: "init a new Launchpad config",
+		Short: "Initialize a new Launchpad config",
```
Comment (LucilleH) [inline padcli/command/initcmd.go:20]: Initializes?

File: padcli/command/version.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ func versionCmd() *cobra.Command {
 
 	var versionCmd = &cobra.Command{
 		Use:   "version",
-		Short: "print the version number",
+		Short: "Print the version number",
```
Comment (LucilleH) [inline padcli/command/version.go:19]: Prints?

**Review by LucilleH — COMMENTED** [review]
nah

**Pull request merged**